utils/verifierFields.ts

```diff
@@ -0,0 +1,12 @@
+import { stringToHex } from "./feltService";
+import { formatHexString } from "./stringService";
+
+export const STARKNET = getFormattedField("starknet");
+export const DISCORD = getFormattedField("discord");
+export const TWITTER = getFormattedField("twitter");
+export const GITHUB = getFormattedField("github");
+export const PROOF_OF_PERSONHOOD = getFormattedField("proof_of_personhood");
+
+function getFormattedField(field: string): string {
```

Comment:
test this (sorry that's the rule lol)

Reply:
This function is not exported and should not be exported. It is just a shortcut on two already tested functions.

Should I export it just to test it? Looks like it will pollute the namespace